README.md

```diff
@@ -161,10 +175,25 @@ pip install .
 ## 2. Sample Sequences Using Monte Carlo
 
 **File**: `sample.py`
-**Description**:  
 
+**Description**:
+  
 Uses Monte Carlo sampling to sample sequences from a  probability distribution. A temperature factor can be applied to affect the distributions. It will return a `.fasta` file and/or a `.json` file with the sequences and a `.csv` file with basic sequence metrics such as isoelectric point, molecular weight and charge. Further metrics can be calculated using NetSolP-1.0 (see `scripts/run_netsolp.sh`).
 
+### 2.1. Generating the input files for Monde Carlo sampling
+
+Run the prediction step using predict.py as instructed in section 1.2.
+
+
+### 2.2 Run:
+
+```
+python sample.py --path_to_pred_matrix /PATH/TO/{MODEL_NAME}.csv --path_to_datasetmap /PATH/TO/{MODEL_NAME}.txt --seed 42 --workers 35 --sample_n 200 --temperature 0.1
+```
```

Comment:
Great! Could you also add an example script to follow on the example from section 1.2? This way people can just copy and paste it.